packages/client/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "@streamr/dev-config/ts/tsconfig.browser.json",
     "compilerOptions": {
         "outDir": "dist",
-        "rootDirs": ["src", "test"]
+        "rootDirs": ["src", "test"],
```

Comment:
@jtakalai @timoxley @harbu @teogeb
Any reason for not using these configuration settings?
```
    "incremental": true,                              /* Enable incremental compilation */
    "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
    "tsBuildInfoFile": ".tsbuildinfo",                /* Specify the folder for .tsbuildinfo incremental compilation files. */
```
Is the `.tsbuildinfo` cacheable in CI?

Reply:
I see `incremental` is set in the config, but it requires `composite = true`

Reply:
no idea